src/cc/bpfd/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) Jazel Canseco, 2018
+# Licensed under the Apache License, Version 2.0 (the "License")
+
+add_executable(bpfd bpfd.c base64.c remote_perf_reader.c utils.c)
+target_link_libraries(bpfd bpf-static)
```

Comment:
I was thinking for the upstream port, we really don't want static, it will raise a red flag.. Can you change it to be dynamic linked for upstream?

Reply:
Done.